earth_enterprise/integration_test/step_impl/minification.py

```diff
@@ -0,0 +1,51 @@
+from getgauge.python import step, before_scenario, Messages
+import os, random, fnmatch, subprocess, datetime
+from subprocess import Popen
+import shutil
+
+# Re-use some of the functionalities
+import assets
+
+
+MISC_XML_PATH = os.path.join(assets.ASSET_ROOT, ".config", "misc.xml")
+
+
+@step("Turn minification <status>")
+def turn_minification(status):
```

Comment:
Minor point: this won't work if `UseMinification` isn't in the misc.xml file to begin with, which is allowed by Fusion. I'd suggest checking if the line is there, using sed to add `<UseMinification>0<\/UseMinification>` right before `</MiscConfigStorage>` if it's not there, then running the existing sed commands as they are.